docs/DeveloperGuide.adoc

```diff
@@ -249,12 +271,65 @@ image::CommitActivityDiagram.png[]
 ** Cons: Requires dealing with commands that have already been undone: We must remember to skip these commands. Violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two different things.
 // end::undoredo[]
 
-// tag::dataencryption[]
-=== [Proposed] Data Encryption
 
-_{Explain here how the data encryption feature will be implemented}_
+=== Expense Splitter feature
+
+The Expense Splitter feature is designed to aid our users with the splitting of large bills that involves meany people.
+The feature comprises of six commands namely.
+
+* <<Add-Item-Command, `AddItemCommand`>> - Creates an Item with its ItemPrice, Person(s) involved in splitting that Item and adds it to Receipt.
+* <<Delete-Item-Command, `DeleteItemCommand`>> - Deletes an Item from the current Receipt and reduces the Person's amount accordingly.
+* <<List-Receipt-Command, `ListReceiptCommand`>> - Lists the Item(s) in the current Receipt, its ItemPrice and Person(s) involved in splitting that Item.
+* <<List-Amount-Command, `ListAmountCommand`>> - Lists the Person(s) Name and Amount they owe the user.
+* <<Paid-Command, `PaidCommand`>> - Reduces the Amount a Person owes.
+* <<Back-Command, `BackCommand`>> - Exits Expense Splitter portion to return to EYLAH's Main Menu.
+
+Below are some diagrams to explain how our team implemented this portion.
+
+
+*Activity Diagram* of Expense Splitter: +
+
+image::activityDiagram_for_expensesplitter.png[]
```

Comment:
For the activity diagram of expense splitter, there should be a branch node for the highlighted arrow. Currently, it is being joined back to the return arrow for delete item.

I think that the branch condition `[Have Item to add?]` should not be phrased as a question. It should be like `Have item to add` instead.

Why is there a `Delete Item` Action for the case when an item is added wrongly? Shouldn't you check for the validity of an added item?
![image](https://user-images.githubusercontent.com/42311220/77503030-85ac4c80-6e97-11ea-976a-1817b883763a.png)